tableaudocumentapi/xfile.py

```diff
@@ -70,6 +70,16 @@ def get_xml_from_archive(filename):
     return xml_tree
 
 
+def create_column(caption, datatype, role, field_type, name):
```

Comment:
@RussTheAerialist wanted this as a column tag factory so it would read 

`column = xml_factory.create_column(caption, datatype, role, type_, name)`

So I put it into xfile, since it's the closest thing to a dedicated XML class that I could find in this project. I'll put it back into fields.py